devel/universal-ctags/Portfile

```
@@ -3,10 +3,10 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        universal-ctags ctags feffe43a4a9d339cd6701744589746e2297e4567
+github.setup        universal-ctags ctags 0837c42c3fea8854499cf1d22f2f5de12d745f20
 name                universal-ctags
-version             20200111
-revision            1
+version             p5.9.20201213.0
```

```
$ ~/macports/macports-user-ryandesign/scripts/vercmp 20200111 p5.9.20201213.0
MacPorts considers 20200111 to be greater than p5.9.20201213.0.
```
Therefore the epoch needs to be increased.
```suggestion
epoch               1
version             p5.9.20201213.0
```
How did you determine that the version number should be p5.9.20201213.0? Specifically, beginning a version number with a letter is unusual. According to the project's configure.ac, its version is 5.9.0. So perhaps 5.9.0-20201213 would be more normal. In fact https://github.com/universal-ctags/ctags/commit/0837c42c3fea8854499cf1d22f2f5de12d745f20 was committed on 20201210. There are newer commits, should you wish to switch to one of them instead and update the date to match.
```suggestion
epoch               1
version             5.9.0-20201210
```

---

Okay, very thanks. default Python version to 3.9.